_release-pypi/action.yml

```
@@ -100,6 +100,12 @@ runs:
   using: "composite"
   steps:
 
+    - name: "Notify to use trusted publisher"
+      shell: ${{ runner.os == 'Windows' && 'powershell' || 'bash' }}
+      if: inputs.use-trusted-publisher == 'false'
+      run: |
+        echo "::notice::We recommend using Trusted Publishers to securely publish packages on PyPI. Please consider moving to this approach."
```

```suggestion
        echo "::notice::We recommend using trusted publishers to securely publish packages on PyPI. You should consider moving to this approach."
```
It doesn't appear that Trusted Publishers should be treated as a proper noun. Avoid "please" in technical documentation (although this does seem to be changing as it is so avoid used in writing prompts for AI)!